packages/office-ui-fabric-react/src/components/ColorPicker/ColorRectangle/ColorRectangle.styles.ts

```diff
@@ -50,7 +50,7 @@ export const getStyles = (props: IColorRectangleStyleProps): IColorRectangleStyl
         width: 20,
         height: 20,
         background: 'white',
-        border: `1px solid ${palette.neutralTertiary}`,
+        border: `1px solid ${palette.white}`,
```

Comment:
I am confused why [this](https://github.com/OfficeDev/office-ui-fabric-react/pull/9116/files#diff-52bd97deee4e3d4d609dc64b405dbbc5R66) is different from this PR.

Reply:
In this one its targeting ColorRectangle's thumb, in the fabric-7 one it is targeting slider thumb

Reply:
Ben updated the design, so this is outdated now

Reply:
Why the difference? Are they not supposed to look the same?

Reply:
As long as they look the same in both places I am ok with it because we discussed it offline

Reply:
They look the same. The styles you mentioned in both PRs are targeting different area.